components/pedigree/resources/src/main/resources/pedigree/model/positionedGraph.js

```diff
@@ -3156,38 +3156,17 @@ define([
 
         //=[position]========================================================================
 
-        displayGraph: function(xcoord, message)
```

Comment:
this is used when debugging layout algorithms (I have som etest code in another epository). I would keep it